documenting.rst

```diff
@@ -60,6 +60,75 @@ Please don't let the material in this document stand between the documentation
 and your desire to help out!
 
 
+.. _building-doc:
+
+Building the documentation
+==========================
+
+.. highlight:: bash
+
+
+When you make changes to the Python documentation, you're going to want to
+view those changes locally.  To build the documentation, follow the
+instructions from one of the sections below.  You can view the documentation
+after building the HTML by pointing a browser at the file
+:file:`Doc/build/html/index.html`.
+
+The toolset used to build the docs is written in Python and is called Sphinx_.
+Sphinx is maintained separately and is not included in this tree.  Also needed
+are docutils_, supplying the base markup that Sphinx uses; Jinja_, a templating
```

Comment:
Docutils is a dependency for Sphinx so I don't think it's worth explaining this much detail (especially if Sphinx ever changes what it depends on).

Reply:
Deferred to a separate PR.